plugins/woocommerce/src/Internal/Admin/EmailPreview.php

```diff
@@ -38,6 +38,19 @@ final public static function instance() {
 		return static::$instance;
 	}
 
+	/**
+	 * Hook into WooCommerce.
+	 */
+	public function __construct() {
+		/**
+		 * Email templates fetch product from the database to show additional information, which are not
+		 * saved in WC_Order_Item_Product. This filter enables fetching those data also in email preview.
```

Comment:
I think "those data" should be "that data".

Reply:
Fixed in cb2bd1c.